framework/decode/vulkan_captured_swapchain.cpp

```diff
@@ -240,37 +240,6 @@ VkResult VulkanCapturedSwapchain::CreateRenderPass2(PFN_vkCreateRenderPass2
     return func(device, create_info, allocator, render_pass);
 }
 
-void VulkanCapturedSwapchain::CmdPipelineBarrier(PFN_vkCmdPipelineBarrier     func,
-                                                 const CommandBufferInfo*     command_buffer_info,
-                                                 VkPipelineStageFlags         src_stage_mask,
-                                                 VkPipelineStageFlags         dst_stage_mask,
-                                                 VkDependencyFlags            dependency_flags,
-                                                 uint32_t                     memory_barrier_count,
-                                                 const VkMemoryBarrier*       memory_barriers,
-                                                 uint32_t                     buffer_memory_barrier_count,
-                                                 const VkBufferMemoryBarrier* buffer_memory_barriers,
-                                                 uint32_t                     image_memory_barrier_count,
-                                                 const VkImageMemoryBarrier*  image_memory_barriers)
-{
-    VkCommandBuffer command_buffer = VK_NULL_HANDLE;
-
-    if (command_buffer_info != nullptr)
-    {
-        command_buffer = command_buffer_info->handle;
-    }
-
-    func(command_buffer,
-         src_stage_mask,
-         dst_stage_mask,
-         dependency_flags,
-         memory_barrier_count,
-         memory_barriers,
-         buffer_memory_barrier_count,
-         buffer_memory_barriers,
-         image_memory_barrier_count,
-         image_memory_barriers);
-}
```

Comment:
This code was really useless and over complicated the swapchain path.  Just removed it here and added the direct function call in the consumer base above.